kfs-core/src/main/java/edu/arizona/kfs/fp/dataaccess/impl/DisbursementVoucherDaoOjb.java

```diff
@@ -0,0 +1,29 @@
+package edu.arizona.kfs.fp.dataaccess.impl;
+
+import java.util.Collection;
+
+import org.apache.ojb.broker.query.Criteria;
+import org.apache.ojb.broker.query.QueryByCriteria;
+
+import edu.arizona.kfs.fp.document.DisbursementVoucherDocument;
+import edu.arizona.kfs.sys.KFSConstants;
+import edu.arizona.kfs.sys.KFSPropertyConstants;
+
+public class DisbursementVoucherDaoOjb extends org.kuali.kfs.fp.dataaccess.impl.DisbursementVoucherDaoOjb {
+    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(DisbursementVoucherDaoOjb.class);
+    private static final String FINANCIAL_DOCUMENT_STATUS_CODE = KFSConstants.DOCUMENT_HEADER_PROPERTY_NAME + "." + KFSPropertyConstants.FINANCIAL_DOCUMENT_STATUS_CODE;
```

Comment:
Just a nit and not worth redoing for, but perhaps using KfsConstants.DELIMITER for the period char?